src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -165,11 +183,27 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
         ListEmptyComponent={renderEmpty ? EmptyOrError : null}
         onEndReachedThreshold={0.5}
       />
+      {showErrorBanner && (
+        <InLineNotification
+          variant={NotificationVariant.Error}
+          title={t('points.history.pageError.title')}
+          description={t('points.history.pageError.subtitle')}
+          ctaLabel={t('points.history.pageError.refresh')}
+          onPressCta={onPressRefresh}
+          withBorder={true}
+          style={styles.errorBanner}
```

Comment:
you'll also need to add something like `{ paddingBottom: Math.max(insets.bottom, Spacing.Thick24) }` here, otherwise this content collides with the bottom bar.

![Simulator Screenshot - iPhone 14 Pro - 2024-04-30 at 09 49 55](https://github.com/valora-inc/wallet/assets/20150449/6df1fc73-6948-4136-b9c5-137554366d45)

Reply:
i didn't notice this before, but it seems like the section headers are sticky? is that expected? the homefeed doesn't do this and i guess it doesn't matter so much but in my screenshot above you can see some clipped content which was unexpected for me

Reply:
Hmm, this actually isn't something I've ever experienced on Android -- I wonder if this is a platform-specific thing?

Reply:
I ran into this on a different screen. https://reactnative.dev/docs/sectionlist#stickysectionheadersenabled - defaults to true on iOS